list.rb

```diff
@@ -0,0 +1,13 @@
+require './my_enumerable'
```

Comment:
* Kindly use `require_relative` to import `MyEnumerable` since using `require_relative` for files in your directory instead of `require` is a ruby convention. "Require should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory." [Read more here ](https://medium.com/@ellishim/understanding-require-vs-require-relative-vs-require-all-80e3b26d89e6#:~:text=require_relative%20is%20a%20subset%20of,within%20the%20same%20project%20directory).&text=Looking%20at%20the%20snippet%20above,option%20when%20calling%20on%20files.)

Reply:
Thanks @Davidosky007 Appreciate your help 🙏